lib/go/test/dontexportrwtest/compile_test.go

```diff
@@ -29,10 +28,10 @@ import (
 func TestReadWriteMethodsArePrivate(t *testing.T) {
 	// This will only compile if read/write methods exist
 	s := NewTestStruct()
-	fmt.Sprintf("%v", s.read)
-	fmt.Sprintf("%v", s.write)
+	_ = s.read
```

Comment:
Nice.. I had done everything else but I had no idea how to fix this in golang.  Thanks!

Reply:
The problem is `s.read` and `s.write` are functions and there just isn't any way to print functions as strings, so I had to find another way to use them without trying to stringify them.